python/agents/brand-search-optimization/deployment/test_deployment.py

```
@@ -0,0 +1,119 @@
+    while True:
+        user_input = input("Input: ")
+        if user_input == "quit":
+            break
+
+        for event in agent.stream_query(
+            user_id=FLAGS.user_id,
+            session_id=session.id,
+            message=user_input
+        ):
+            if "content" in event:
+                if "parts" in event["content"]:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)